### DISCUSSION TITLE
Documentation is outdated

### DISCUSSION THREAD

**Discussion by shikida** [ical4j/ical4j#753] [Q&A]
I am trying to use the library to create a couple of new ics files but I've noticed that the site documentation and examples are quite outdated, full of deprecated classes.

Is there any plan to provide an up to date documentation with examples that are not using any deprecated code?

TIA

**Comment by shikida** [comment]
Please also consider keeping the whole project documentation in github as well

**Reply by benfortuna** [reply]
It is: https://github.com/ical4j/ical4j-user-guide

Feel free to fork and contribute updates via pull requests as you see fit.
